runelite-client/src/main/java/net/runelite/client/plugins/varrockmuseum/VarrockMuseumAnswer.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2018, Viktor <Viktor.horsmanheimo@teknik.io>
+ * Copyright (c) 2018, Tomas Slusny <slusnucky@gmail.com>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.varrockmuseum;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.Client;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetInfo;
+
+@Getter
+@RequiredArgsConstructor
+enum VarrockMuseumAnswer
+{
+
+	LIZARD_1("How does a lizard regulate body heat?", "Sunlight."),
+	LIZARD_2("Who discovered how to kill lizards?", "The Slayer Masters."),
+	LIZARD_3("How many eyes does a lizard have?", "Three."),
+	LIZARD_4("What order do lizards belong to?", "Squamata."),
+	LIZARD_5("What happens when a lizard becomes cold?", "It becomes sleepy."),
+	LIZARD_6("Lizard skin is made of the same substance as?", "Hair."),
+
+	TORTOISE_1("What is the name of the oldest tortoise ever recorded?", "Mibbiwocket."),
+	TORTOISE_2("What is a tortoise's favourite food?", "Vegetables."),
+	TORTOISE_3("Name the explorer who discovered the world's oldest tortoise.", "Admiral Bake."),
+	TORTOISE_4("How does the tortoise protect itself?", "Hard shell."),
+	TORTOISE_5("If a tortoise had twenty rings on its shell, how old would it be?", "Twenty years."),
+	TORTOISE_6("Which race breeds tortoises for battle?", "Gnomes."),
+
+	DRAGON_1("What is considered a delicacy by dragons?", "Runite."),
+	DRAGON_2("What is the best defence against a dragon's attack?", "Anti-dragon-breath shield."),
```

Comment:
Remove '-' between Anti and dragon
![DRAGON_2](https://i.imgur.com/YzhKFne.png)